src/resources/help/en_US/relnotes3.3.0.html

```diff
@@ -145,6 +145,7 @@
   <li>Fixed a bug with the Python scripting interface where the PySide2 module wouldn't load when running one of the pre-built binaries.</li>
   <li>The Contour plot's fields were re-ordered internally to fix a bug where a command-recorded script of a Contour plot would crash the Python scripting interface if the number of contours had been increased from the default.</li>
   <li>The images and wording for <i>Curve Overlay</i> and <i>Reflected Curve Overlay</i> templates in the Save Movie Wizard were updated to demonstrate that they can be used on 3D plots, not just 2D plots.</li>
+  <li>Fixed a bug with the color table window where the list of color tables could be populated even when no tags were selected.</li>
```

Comment:
Should check with Eric if he will be re-doing the 3.3 builds to get this change. If not, this should go in 3.3.1 release notes.

Reply:
I don't plan on re-doing the 3.3 builds, so yes, you will need to create a 3.3.1 release notes file and put the change in there.

Reply:
Let me know if the new 3.3.1 relnotes file looks good.